### ISSUE TITLE
Share function not working (urgent issue)

### ISSUE THREAD

**Issue by MJWainwright** [epistemonikos/isoq#31]
A user is trying to share the project with co-reviewers and it is not working. She put all three in at once, separated by commas, clicked add and then invite. But nothing happened. Only the first address appeared in "Users with access" under pending access. However, that user reports that they never received an email. We repeatedly tried entering the emails one by one and trying to click add and then invite, but nothing was happening. She is due to begin working with the team next week so resolving this is quite urgent

**Comment by damian-garrido** [comment]
@MJWainwright this issue is still alive?

**Comment by MJWainwright** [comment]
It has not happened again. I'm not sure we found the problem though. Users did recently however find it impossible to login until they cleared all their cache memory. I was able to login with their credentials but they were not. I asked them to clear all their browser history and cache and then they could login to see the shared project. One thought that this was because she had initially asked google to remember her password but then subsequently changed she password. I will keep monitoring this issue and let you know if it happens again
